src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -141,7 +91,7 @@ private static IEnumerable<ProjectContext> GetMatchingProjectContexts(IEnumerabl
         /// <param name="outputPath">Location of published files</param>
         /// <param name="configuration">Debug or Release</param>
         /// <returns>Return 0 if successful else return non-zero</returns>
-        private static int Publish(ProjectContext context, string outputPath, string configuration)
+        private static int PublishRidFxPermutation(ProjectContext context, string outputPath, string configuration)
```

Comment:
True although it is what exactly this is doing in this context. Probably PublishProjectContext is equally accurate and even slightly better. What about PublishProjectContext and a comment "in this context ProjectContext represents single Rid & Fx permutation" - that would make it clear and easy to understand. Thoughts?